projects/dsp-ui/src/lib/search/expert-search/expert-search.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, DebugElement, OnInit, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ActivatedRoute } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { KnoraApiConfig } from '@dasch-swiss/dsp-js';
+import { DspApiConfigToken } from '../../core/core.module';
+import { ExpertSearchComponent } from './expert-search.component';
+
+/**
+ * Test host component to simulate parent component.
+ */
+@Component({
+    template: `
+        <dsp-expert-search #expSearch></dsp-expert-search>`
+})
+class TestHostComponent implements OnInit {
+
+    @ViewChild('expSearch') expertSearch: ExpertSearchComponent;
+
+    ngOnInit() {
+    }
+
+}
+
+describe('ExpertSearchComponent', () => {
+  let testHostComponent: TestHostComponent;
+  let testHostFixture: ComponentFixture<TestHostComponent>;
+  let hostCompDe: DebugElement;
+
+  const dspConfSpy = new KnoraApiConfig('http', 'localhost', 3333, undefined, undefined, true);
```

Comment:
Move this into the `beforeEach`. I think here it does not make a difference, but it is a good practice.

If there are side-effects (setting a token etc.), then this could lead to unpredictable behaviour since the `it`s are run in random order.

Reply:
done in 062ccea

Reply:
ok